osu.Game/Beatmaps/IWorkingBeatmap.cs

```diff
@@ -57,9 +58,10 @@ public interface IWorkingBeatmap
         /// <param name="ruleset">The <see cref="RulesetInfo"/> to create a playable <see cref="IBeatmap"/> for.</param>
         /// <param name="mods">The <see cref="Mod"/>s to apply to the <see cref="IBeatmap"/>.</param>
         /// <param name="timeout">The maximum length in milliseconds to wait for load to complete. Defaults to 10,000ms.</param>
+        /// <param name="cancellationToken">Externally provided cancellation token.</param>
```

Comment:
kind of pointless xmldoc - would be better to explain what it does (i.e. that it cancels the process of loading the beatmap)